srcpkgs/wingpanel/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'wingpanel'
+pkgname=wingpanel
+version=2.3.0
+revision=1
+build_style=meson
+hostmakedepends="cmake pkg-config glib-devel"
+makedepends="gnome-desktop-devel vala-devel gtk+3-devel libgee08-devel granite-devel mutter-devel gala-devel"
```

Comment:
These dependencies are wrong and/or overly verbose. Please just use the commit/template from #20429 as-is.